lib/node_modules/@stdlib/blas/ext/base/dnansumors/lib/ndarray.js

```diff
@@ -30,8 +30,8 @@ var isnan = require( '@stdlib/math/base/assert/is-nan' );
 *
 * @param {PositiveInteger} N - number of indexed elements
 * @param {Float64Array} x - input array
-* @param {integer} stride - stride length
-* @param {NonNegativeInteger} offset - starting index
+* @param {integer} strideX - indexing increment
```

Comment:
```suggestion
* @param {integer} strideX - index increment
```

Reply:
This change is likely not necessary given that `x.native.js` using `stride length` for the description.

Reply:
I am fine standardizing around a particular convention, but we should be consistent(ish) within a package.